model/godleyTableWindow.cc

```diff
@@ -837,12 +839,16 @@ namespace minsky
     historyPtr-=changes;
     if (historyPtr > 0 && historyPtr <= history.size())
       {
-        auto& d=history[historyPtr-1];
+        auto& d=history[historyPtr-1].first;
```

Comment:
If we make the change above to stashing full GodleyTable objects, then this bit of code becomes simple assignment.

Reply:
I took me quite a while to get the above to work, so again, thanks for the suggestion.  It will be a  boon to simplify the code to something similar to the original, in terms of keeping Minsky streamlined and efficient too.